### ISSUE TITLE
Can't create nullable timestamps

### ISSUE THREAD

**Issue by 1RV34** [phalcon/cphalcon#13099]
### Expected and Actual Behavior

I'm trying to use the `Column` class to create tables in my database. In these tables I try to create a timestamp column that is nullable using the following code:
```php
new Column('deadline', 'type' => Column::TYPE_TIMESTAMP, 'notNull' => false, 'default' => null)
```
I would expect to have created a nullable timestamp column with its default value being `null`, however instead a timestamp column would be created with `current_timestamp` as default together with the `on update current_timestamp` property, all following timestamp columns will have default `'0000-00-00 00:00:00'`.

### Details

* Phalcon version: 3.2.1
* PHP Version: 7.1.9
* Operating System: ubuntu 16.04.1
* Installation type: Compiling from source
* Zephir version: 0.9.9
* Server: Apache
* Other related info:
    Generated query will have the following in it:
    ```sql
    CREATE TABLE `table` (
        `timestamp1` timestamp,
        `timestamp2` timestamp
    );
    ```
    which will result in a table with columns like:
    ```sql
    CREATE TABLE `table` (
        `timestamp1` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURENT_TIMESTAMP,
        `timestamp2` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00'
    );
    ```
    Instead it should generate:
    ```sql
    CREATE TABLE `table` (
        `timestamp1` timestamp NULL DEFAULT NULL,
        `timestamp2` timestamp NULL DEFAULT NULL
    );
    ```

**Comment by sergeyklay** [comment]
As you can see tests failed

> [`Failures: 40`](https://travis-ci.org/phalcon/cphalcon/jobs/281230808#L2723)

**Comment by 1RV34** [comment]
I saw, working on updating the tests.

**Comment by 1RV34** [comment]
Do you think this fix could be included with **3.2.x**, since it's a bugfix not a feature? (Current Phalcon system doesn't respond to `'notNull' => false` correctly and causes issues. It's causing our applications to break.)

**Comment by 1RV34** [comment]
Sorry, wrong button.
